src/ngx_http_lua_kong_common.h

```diff
@@ -34,6 +34,16 @@ typedef struct {
     ngx_str_t           tag;
 } ngx_http_lua_kong_loc_conf_t;
 
+typedef struct {
+    /* fields for dynamic log level */
+    ngx_uint_t dynmic_log_level;            /* the dynamic log level */
```

Comment:
curious, do we have a typo `dynmic_log_level` or missing the `a` was it intentional?

Reply:
fixed